### ISSUE TITLE
 cannot load such file -- /opt/postal/app/lib/postal/version

### ISSUE THREAD

**Issue by dani** [postalserver/postal#2954]
## Describe the bug

When trying to run postal update (or postal version), it fails with this error
```
postal@a2e543e648e0:~/app$ postal version
<internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require': cannot load such file -- /opt/postal/app/lib/postal/version (LoadError)
	from <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
	from script/version.rb:4:in `<main>'
postal@a2e543e648e0:~/app$ 
```

This is reproducible on the official Docker image, as well as in my custom, alpine based image.

## To Reproduce

1. Run the docker image with eg docker run --rm -it ghcr.io/postalserver/postal:3.3.3 postal version
3. See the error

## Expected behaviour

Command should not fail (obviously :-) )

## Environment details 

 - Docker 26.1.1
 - Official Docker image
 - 3.3.3

**Comment by dani** [comment]
The issue started with 3.3.2. It's working in 3.3.1

**Comment by bonest** [comment]
Can confirm the same issue on running postal version.

`<internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require': cannot load such file -- /opt/postal/app/lib/postal/version (LoadError)
	from <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
	from script/version.rb:4:in `<main>'`

**Comment by ugurtam** [comment]
Yes same issue

**Comment by adamcooke** [comment]
This is fixed in 3.3.4.
